test/chi0.jl

```diff
@@ -104,15 +110,19 @@ end
     for (case, temperatures) in [(silicon, (0, 0.03)), (magnesium, (0.01, ))]
         for temperature in temperatures, spin_polarization in (:none, :collinear)
             for symmetries in (false, true)
-                test_chi0(case; symmetries, temperature, spin_polarization)
+                for fix_εF in (false, true)
+                    test_chi0(case; symmetries, temperature, spin_polarization, fix_εF)
+                end
```

Comment:
Not sure about the full multiplication by 2 in the number of tests. Is it not enough to test some of the combinations here? E.g. metal with collinear / non-collinear spin and fixed fermi level?